pkg/auth/auth.go

```diff
@@ -153,16 +155,45 @@ func Login(ctx context.Context, systemContext *types.SystemContext, opts *LoginO
 	return errors.Wrapf(err, "authenticating creds for %q", server)
 }
 
-// getRegistryName scrubs and parses the input to get the server name
-func getRegistryName(server string) string {
+// getCredentials returns the auth config for the provided URL.
+func getCredentials(systemContext *types.SystemContext, url string) (res types.DockerAuthConfig, err error) {
```

Comment:
The parameter is not an URL. (Also, to an extent, elsewhere.)

Reply:
Fixed